packages/wrangler/src/__tests__/get-entry.test.ts

```diff
@@ -0,0 +1,128 @@
+import { writeFile } from "fs/promises";
+import path from "path";
+import dedent from "ts-dedent";
+import { Config } from "../config";
+import { defaultWranglerConfig } from "../config/config";
+import { Entry, getEntry } from "../deployment-bundle/entry";
+import guessWorkerFormat from "../deployment-bundle/guess-worker-format";
+import { mockConsoleMethods } from "./helpers/mock-console";
+import { runInTempDir } from "./helpers/run-in-tmp";
+import { seed } from "./helpers/seed";
+
+function normalize(entry: Entry): Entry {
+	return JSON.parse(
+		JSON.stringify(entry).replaceAll(process.cwd(), "/tmp/dir")
+	);
+}
+describe("getEntry()", () => {
+	runInTempDir();
+	mockConsoleMethods();
+	it.each([
```

Comment:
I love that you added these tests, but if I am honest, using `it.each()` here actually obscures what the tests are doing and doesn't save much boilerplate compared to writing out a bunch of separate it clauses, given how simple the test code is.